content/locales/en.ts

```diff
@@ -36,6 +36,13 @@ const addressInputStrings = {
   cancel: "Cancel",
 };
 
+const identitiesInputStrings = {
+  emailPlaceholder: "frieda@ewlines.tech",
+  phonePlaceholder: "01632 960049",
```

Comment:
Is it a random fake value ? Or could it be a real phone number somewhere ?

Reply:
This is a fake one 🙂.

Reply:
The English address is also a fake one.